libs/filamat/include/filamat/MaterialBuilder.h

```diff
@@ -525,6 +525,9 @@ class UTILS_PUBLIC MaterialBuilder : public MaterialBuilderBase {
     //! Specify the stereoscopic type (default is INSTANCED)
     MaterialBuilder& stereoscopicType(StereoscopicType stereoscopicType) noexcept;
 
+    //! Specify the number of eye count for stereoscopic rendering
```

Comment:
nit:
```suggestion
    //! Specify the number of eyes for stereoscopic rendering
```

Maybe also document the default number of eyes.